Client/Application/AppDelegate.swift

```diff
@@ -294,6 +368,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
                 url = item.value
             case "private":
                 isPrivate = NSString(string: item.value ?? "false").boolValue
+            case "deep-link":
+                guard let value = item.value else { break }
+                Router.shared.routeURL(value)
```

Comment:
Since we require being on the main thread per the asserts, should the routeURL call be explicitly placed on the main thread?